RiotNSE/NotificationService.swift

```diff
@@ -27,8 +27,8 @@ class NotificationService: UNNotificationServiceExtension {
     
     /// Cached events. Keys are eventId's
     var cachedEvents: [String: MXEvent] = [:]
-    var mxSession: MXSession?
-    var store: NSEMemoryStore!
+    static var mxSession: MXSession?
+    static var store: NSEMemoryStore!
```

Comment:
Do we need `store` to be static?
Do we need to keep a reference to it? We have `mxSession.store`

Reply:
Sure we don't, updating